bccsp/pkcs11/pkcs11.go

```diff
@@ -93,13 +94,15 @@ func (csp *impl) getSession() (session pkcs11.SessionHandle) {
 func createSession(ctx *pkcs11.Ctx, slot uint, pin string) pkcs11.SessionHandle {
 	var s pkcs11.SessionHandle
 	var err error
+	// attempt 10 times to open a session with a 100ms backoff after each attempt
```

Comment:
nit: it's a delay instead of a backoff.

Reply:
I'll make the change since CI is still running anyways